deploy/helm/kubecf/assets/operations/instance_groups/app-autoscaler.yaml

```diff
@@ -0,0 +1,1040 @@
+              - http://127.0.0.1:6205
+- type: replace
+  path: /instance_groups/name=asnozzle/jobs/name=metricsgateway/properties/quarks?
+  value:
+    ports:
+    - name: mghealthport
+      portocol: TCP
+      internal: 6503
+    bpm:
+      processes:
+      - name: metricsgateway
+        executable: /var/vcap/packages/metricsgateway/metricsgateway
+        args:
+        - -c
+        - /var/vcap/jobs/metricsgateway/config/metricsgateway.yml
+    run:
+      healthcheck:
+        metricsgateway:
+          readiness:
+            exec:
+              command:
+              - curl
+              - --fail
+              - --head
+              - http://127.0.0.1:6503
+- type: replace
+  path: /instance_groups/name=asapi/jobs/name=golangapiserver/properties/quarks?
+  value:
+    ports:
+    - name: golangapiserver
+      protocol: TCP
+      internal: *apiServerPublicPort
+    - name: apihealthport
+      portocol: TCP
+      internal: 1080
+    - name: brokerport
+      portocol: TCP
+      internal: 6102
+    bpm:
+      processes:
+      - name: golangapiserver
+        executable: /bin/bash
+        args:
+        - -c
+        - |
+          /var/vcap/jobs/golangapiserver/bin/apiserver_ctl start
+    run:
+      healthcheck:
+        golangapiserver:
+          readiness:
+            exec:
+              command:
+              - curl
+              - --fail
+              - --head
+              - http://127.0.0.1:1080
+- type: replace
+  path: /instance_groups/name=asapi/jobs/name=metricsforwarder/properties/quarks?
+  value:
+    ports:
+    - name: metricsforwarder
+      protocol: TCP
+      internal: 6201
+    - name: mgfdhealthport
+      portocol: TCP
+      internal: 6403
+    bpm:
+      processes:
+      - name: metricsforwarder
+        executable: /bin/bash
+        args:
+        - -c
+        - |
+          /var/vcap/jobs/metricsforwarder/bin/metricsforwarder_ctl start
+    run:
+      healthcheck:
+        metricsforwarder:
+          readiness:
+            exec:
+              command:
+              - curl
+              - --fail
+              - --head
+              - http://127.0.0.1:6403
+- type: replace
+  path: /instance_groups/name=postgres_autoscaler/jobs/name=postgres/properties/quarks?
+  value:
+    ports:
+    - name: postgres
+      protocol: TCP
+      internal: *postgresPort
+    run:
+      healthcheck:
+        postgres:
+          readiness:
+            exec:
+              command:
+              - /var/vcap/packages/postgres-11.5/bin/pg_isready
+    pre_render_scripts:
+      jobs:
+      - |
+        sed -i "s/sysctl/#sysctl/g" /var/vcap/all-releases/jobs-src/postgres/postgres/templates/pre-start.sh.erb
+    bpm:
+      processes:
+      - name: postgres
+        limits:
+          open_files: 1048576
+        persistent_disk: true
+        executable: /usr/bin/su
+        args:
+        - "-"
+        - vcap
+        - -c
+        - |
+          set -o errexit -o nounset
```

Comment:
… shouldn't we patch this script into existence, rather than having it live inside an ops file?

Reply:
Not sure where the patch script to put..

Reply:
We have a bunch of [pre-render scripts](https://github.com/SUSE/kubecf/tree/master/bosh/releases/pre_render_scripts); for example, there's one to [patch UAA startup](https://github.com/SUSE/kubecf/blob/master/bosh/releases/pre_render_scripts/uaa/uaa/jobs/patch_uaa.sh).  We might be able to do something similar here (but possibly just `cat > /var/vcap/jobs/postgres/bin/kubecf-startup.sh <<EOF` or something).

Reply:
We cannot have it there because we don't have a mechanism to make those conditional like autoscaler is.

Reply:
:crying_cat_face: